modules/300-prometheus/openapi/config-values.yaml

```diff
@@ -17,6 +17,8 @@ properties:
 
       `storageClass: false` — forces the `emptyDir` usage. You will need to delete the old PVC and restart the Pod manually.
 
+      **CAUTION!** When migrating Prometheus with local storage to other nodes, the pod will hang in the Pending state. In this case, it will be necessary to save the Prometheus database, delete the old PVC and restart it manually.  Local storage refers to the StorageClass associated with the local volume on the node, not the network storage (for example, StorageClass created by the local-path-provider module).
```

Comment:
```suggestion
      **Caution.** When migrating Prometheus with local storage to other nodes, the pod will hang in the Pending state. In this case, it will be necessary to save the Prometheus database, delete the old PVC and restart the pod manually.  Local storage refers to a StorageClass that is associated not with network storage, but with a local volume on a node (for example, StorageClass created by the local-path-provider module).
```